package.json

```diff
@@ -86,7 +88,10 @@
 		"@wordpress/npm-package-json-lint-config": "file:packages/npm-package-json-lint-config",
 		"@wordpress/postcss-themes": "file:packages/postcss-themes",
 		"@wordpress/scripts": "file:packages/scripts",
+		"babel-jest": "24.8.0",
```

Comment:
Seems like `babel-jest` is a dependency of `@wordpress/jest-preset-default` so it should not be needed as well. Same for `enzyme-adapter-react-16` and `enzyme-to-json`

Reply:
Right! Removed those too and still everything working 👍 
This didn't modify `package-lock.json` either.